app/src/actions/auth.js

```diff
@@ -20,11 +17,30 @@ export function checkLogin() {
     // Add callback for lock's `authenticated` event
     authService.lock.on('authenticated', (authResult) => {
       authService.lock.getProfile(authResult.idToken, (error, profile) => {
-        if (error)
-          return dispatch(loginError(error))
-        AuthService.setToken(authResult.idToken) // static method
-        AuthService.setProfile(profile) // static method
-        return dispatch(loginSuccess(profile))
+        if (error) { return dispatch(loginError(error)) }
+        let userID = profile.identities[0].user_id
+        let newUser = {
+          firstName: profile.given_name,
+          lastName: profile.family_name,
+          email: profile.email,
+          fbID: userID,
+        }
+        console.log('new user', newUser)
+        axios
+          .get(`http://localhost:3000/users?authID=${userID}`)
+          .then((response) => {
+            console.log('response in axios get request', response)
+            if (!response.data.length) {
+              axios.post('http://localhost:3000/users', newUser)
```

Comment:
We've hard-coded here. 

Let's merge all these together, then open a new PR for refactoring ALL references to the API (heroku OR localhost) into a variable that's loaded via .env